.cci.jenkinsfile

```diff
@@ -47,6 +47,7 @@ parallel insttests: {
           (cd overrides/rpm && curl -LO https://kojipkgs.fedoraproject.org//packages/libsolv/0.7.17/1.fc33/x86_64/libsolv-0.7.17-1.fc33.x86_64.rpm)
           coreos-assembler fetch
           coreos-assembler build
+           ${env.WORKSPACE}/ci/composepost-checks.sh
```

Comment:
I looked at adding this to fcos-e2e but it's tricky because this test is actually a new special case; it's not a component or a kola test.  We could add this into our `Dockerfile` flow but we'd need to ship `libtest-core.sh` too.